proof/invariant-abstract/ARM_HYP/ArchVCPU_AI.thy

```diff
@@ -0,0 +1,354 @@
+(*
+ * Copyright 2022, ???
```

Comment:
FIXME copyright

Reply:
Well spotted. In this case UNSW, unless Mitch had already created the file, the D61 and UNSW